codespell_lib/data/dictionary.txt

```diff
@@ -11083,6 +11083,10 @@ decrasing->decreasing, deceasing,
 decration->decoration
 decreace->decrease
 decreas->decrease
+decremeant->decrement
```

Comment:
Add `decremeants`?

Reply:
Thanks a lot for the suggestion. Added in 28d466750b178824e86dd2bdbf0ba82f69ab465b